_haikus/happy_learning.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: happy_learning
+author: Prasanth
+---
+
+Cloud is very cool <br>
+I'm learning how to do stuff <br>
+I am going to win <br>
```

Comment:
This looks great! The only thing that needs to change is the last line as it is one syllable too many. You can try:

```
I'm going to win <br>
```

Reply:
> This looks great! The only thing that needs to change is the last line as it is one syllable too many. You can try:
> 
> ```
> I'm going to win <br>
> ```

taken care :)